Cargo.toml

```diff
@@ -109,7 +109,7 @@ edition = "2021"
 
 [profile.dev]
 panic = "abort"
-lto = false
+lto = true
```

Comment:
What does this do? How does this relate to 
```
  # Enable link-time-optimization
  "-C", "lto",
```
?

Reply:
`"-C", "lto"` is redundant and should be removed. If you set `lto = false` but then pass `"-C", "lto"`, you get a compile error on debug builds.